nixos/modules/services/networking/ddclient.nix

```diff
@@ -102,6 +102,22 @@ in
           Method to determine the IP address to send to the dynamic DNS provider.
         '';
       };
+
+      verbose = mkOption {
+        default = true;
```

Comment:
It was activated by default, so I didn't want to change the current default, but the ability to deactivate it.